lib/SessionVerifier.php

```diff
@@ -145,7 +145,9 @@ public function afterLogout($accessToken, $idToken): void {
 			$this->session->remove('oca.openid-connect.refresh-token');
 			$this->session->remove('oca.openid-connect.id-token');
 			$this->logger->debug('OIDC Logout: ending session ' . $accessToken . ' id: ' . $idToken);
-			$this->client->signOut($idToken, null);
+			$openIdConfig = $this->client->getOpenIdConfig();
+			$redirectUri = $openIdConfig['post_logout_redirect_uri'] ?? null;
```

Comment:
isset ? the key might be completly missing - right?

Reply:
https://www.tutorialspoint.com/php7/php7_coalescing_operator.htm ;)